docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -552,3 +552,15 @@ v1.2.20, pending the resolution of this issue. We will reintroduce them once we
 determine the appropriate action to address this dependency. Progress on this
 matter is being monitored through issue
 [288](https://github.com/Dasharo/dasharo-issues/issues/288).
+
+### Verify Boot Guard key
+
+It's possible to verify if the currently running firmware is signed using the
```

Comment:
Not **if** the current firmware is signed, the question is: with **which** keys, right?

Reply:
https://github.com/Dasharo/docs/pull/1190/commits/e0c2fa204b5a8be660a8b131ad109b23992f1e0f